bees/instapaperbee/instapaperbee.go

```diff
@@ -0,0 +1,78 @@
+/*
+ *    Copyright (C) 2019 Adam Petrovic
+ *
+ *    This program is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU Affero General Public License as published
+ *    by the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *    Authors:
+ *      Adam Petrovic <adam@petrovic.com.au>
+ */
+
+package instapaperbee
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/muesli/beehive/bees"
+)
+
+type InstapaperBee struct {
+	bees.Bee
+
+	username string
+	password string
+}
+
+func (mod *InstapaperBee) Run(cin chan bees.Event) {
```

Comment:
I think you can omit this function, as that's the standard implementation embedded by `bees.Bee`.